cloudapi/insights/client.go

```diff
@@ -58,6 +63,21 @@ type ClientTLSConfig struct {
 	CertFile string
 }
 
+// RetryConfig is the configuration for the client retries.
+type RetryConfig struct {
```

Comment:
nit: Prefix with `Client` to match other struct names.

Reply:
Done, thanks!